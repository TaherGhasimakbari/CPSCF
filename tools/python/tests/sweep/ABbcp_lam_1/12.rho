 format  1  0
dim                 
                   1
crystal_system      
          'lamellar'
N_cell_param        
                   1
cell_param          
    1.4764149272E+00
group_name          
                '-1'
N_monomer           
                   2
N_star              
                  17
  3.800000002235E-01  6.199999997765E-01       0     1
  2.913683899884E-01 -2.913683899671E-01       1     2
  5.954815289452E-02 -5.954815289978E-02       2     2
 -8.574871643014E-03  8.574871640597E-03       3     2
 -8.294304201552E-03  8.294304207820E-03       4     2
 -1.250930765255E-03  1.250930768392E-03       5     2
  6.125775239091E-04 -6.125775250072E-04       6     2
  3.085377612552E-04 -3.085377627211E-04       7     2
  1.254454577239E-05 -1.254454607682E-05       8     2
 -3.216399439910E-05  3.216399458381E-05       9     2
 -1.011151406675E-05  1.011151416867E-05      10     2
  9.989552109837E-07 -9.989552193000E-07      11     2
  1.448652522223E-06 -1.448652528573E-06      12     2
  2.765074651440E-07 -2.765074291403E-07      13     2
 -9.352732306242E-08  9.352735748049E-08      14     2
 -5.326067376116E-08  5.326065672353E-08      15     2
 -6.779558359149E-09  6.779523997746E-09      16     1
