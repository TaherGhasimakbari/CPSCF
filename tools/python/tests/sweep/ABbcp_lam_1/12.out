 format  1  0

MONOMERS            
N_monomer           
                   2
kuhn                
    1.0000000000E+00    1.0000000000E+00

CHAINS              
N_chain             
                   1
N_block             
                   2
block_monomer       
                   1                   2
block_length        
    3.8000000022E-01    6.1999999978E-01

COMPOSITION         
ensemble            
                   0
phi_chain           
    1.0000000000E+00

INTERACTION         
interaction_type    
               'chi'
chi                 
    1.5000000011E+01

UNIT_CELL           
dim                 
                   1
crystal_system      
          'lamellar'
N_cell_param        
                   1
cell_param          
    1.4764149272E+00

DISCRETIZATION      
ngrid               
                  32
chain_step          
    1.0000000000E-02

FILE_PREFIXES       
input_prefix        
               'in.'
output_prefix       
              'ref/'

BASIS               
group_name          
                '-1'

ITERATE             
max_itr             
                  20
error_max           
    1.0000000000E-08
domain              
                   T
itr_algo            
                'NR'
N_cut               
                  17

SWEEP               
s_max               
    2.0000000000E+01
d_block_length      
   -1.0000000000E-02    1.0000000000E-02
end_increments      

FINISH              

THERMO              
f_Helmholtz         
    2.3656880164E+00
f_homo              
    2.5340000034E+00
pressure            
    3.2052117772E+00
mu_chain            
    5.5708997937E+00
stress              
   -4.2623136804E-11

STATISTICS          
N_star              
                  17
Final Error         
    4.2623136804E-09
Iterations          
                   7
Basis Time          
    2.6167106628E-01
SCF Time            
    5.2084922791E-02
